Scripts/whitesource-spdx.yml

```diff
@@ -0,0 +1,56 @@
+name: WhiteSource scan and SPDX report 
+
+on: 
+  push:
+    branches: []
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
+      - name: Build with Maven
+        run: mvn clean install -DskipTests=true
+
+      - name: WhiteSource Unified Agent Scan
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          WS_APIKEY: ${{secrets.APIKEY}}
+          WS_USERKEY: ${{secrets.USERKEY}}
+          WS_WSS_URL: https://saas.whitesourcesoftware.com/agent
+          WS_PRODUCTNAME: GH_${{github.event.repository.name}}
+          WS_PROJECTNAME: ${{github.ref}}_SPDX_report
+          WS_GENERATEPROJECTDETAILSJSON: true
+        run: |
+          curl -LJO https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar
+          echo Unified Agent downloaded successfully
+          java -jar wss-unified-agent.jar -d ./
+
+      - name: Generate WhiteSource SPDX Report
```

Comment:
Hi , @garymsegal-ws @samq-ws 

Before running this step , the script should check that the scan `requestState` is updated / finished so that the ws_sbom_generator won't run on an empty project.
I want to recommend on this yml in this [issue](https://github.com/whitesource-ps/ws-sbom-generator/issues/80) , but such validation is missing.
 
There is a [validation](https://github.com/whitesource-ps/ws-sbom-generator/blob/ccb5ae6b0eb2ff7d0b166a7f02f79af84a5bedcc/ws_sbom_generator/sbom_generator.py#L72) for empty project , but its relevant for case where no libraries were resolved from the scan.

![image](https://user-images.githubusercontent.com/41573396/155512540-07ed3689-bbb3-405d-9efb-4ef10b05872c.png)